packages/server/nango-integrations/.nango/schema.json

```diff
@@ -0,0 +1,136 @@
+{
```

Comment:
No need to commit the files in `.nango` folder

Reply:
Yeah, removed.